web/src/engine/websites/CoroCoro.ts

```diff
@@ -25,4 +63,69 @@ export default class extends DecoratableMangaScraper {
     public override get Icon() {
         return icon;
     }
+
+    public override async FetchMangas(provider: MangaPlugin): Promise<Manga[]> {
+        const scripts = await FetchCSS<HTMLScriptElement>(new Request(new URL('/rensai', this.URI)), 'script');
+        const json = this.FindAndParseJSON(scripts, 'totalChapterLikes');
+        const { weekdays: { mon, tue, wed, thu, fri, sat, sun } } = this.LocateObjectWithNamedProperty<JSONMangas>(json, 'weekdays');
+        return [mon, tue, wed, thu, fri, sat, sun].reduce((accumulator: Manga[], day) => {
```

Comment:
No need to write each key, consider using `Object.values(weekdays).reduce` instead